examples/set-namespace/.expected/diff.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/configs/example-config.yaml b/configs/example-config.yaml
+index c3ca9ba..6dc6427 100644
+--- a/configs/example-config.yaml
++++ b/configs/example-config.yaml
+@@ -1,20 +1,21 @@
+ apiVersion: v1
++data:
```

Comment:
there's diff here due to YAML formatting that's not related to this function. Please run 'kpt cfg fmt' on the input package first.

Reply:
`set-namespace` function uses the library from kustomize/api and the output mapping order doesn't match the result of `kpt cfg fmt`. IMO we should use the real output from the diff.

Reply:
It's a little hard to read when the diff contains reordering.
I wonder if we should run `kpt cfg fmt` as part of `kpt fn` to enforce the order.

Reply:
From the test's perspective, we don't really care about the content in `diff.patch`. What we really care is the diff between expected diff and actual diff. So the readability in `diff.patch` is not too important here.

+1 for running `fmt` in `fn run`. Will consider it in the pipeline implementation.